assets/package.json

```
@@ -1,5 +1,5 @@
 {
   "dependencies": {
-    "@supabase/realtime-js": "^2.1.0"
+    "@supabase/supabase-js": "^2.21.0"
```

@chasers since you're already version bumping might as well version bump to the latest:

```suggestion
    "@supabase/supabase-js": "^2.24.0"
```